q2_feature_table/plugin_setup.py

```diff
@@ -509,14 +509,17 @@
 plugin.visualizers.register_function(
     function=q2_feature_table.tabulate_seqs,
     inputs={'data': FeatureData[Sequence | AlignedSequence],
-            'taxonomy': FeatureData[Taxonomy]},
-    parameters={'metadata': Metadata},
+            'taxonomy': List[FeatureData[Taxonomy]]},
```

Comment:
Actually make this `Collection[FeatureData[Taxonomy]]` that way they can key the taxonomies if they want.